src/libs/ReportUtils.ts

```diff
@@ -3654,6 +3654,17 @@ function addDomainToShortMention(mention: string): string | undefined {
     return undefined;
 }
 
+function completeShortMention(text: string): string {
+    return text.replace(CONST.REGEX.SHORT_MENTION, (match) => {
+        if (!Str.isValidMention(match)) {
+            return match;
+        }
+        const mention = match.substring(1);
+        const mentionWithDomain = addDomainToShortMention(mention);
+        return mentionWithDomain ? `@${mentionWithDomain}` : match;
+    });
+}
+
```

Comment:
@bernhardoj Can you add comments to this function?

Reply:
Done, please check